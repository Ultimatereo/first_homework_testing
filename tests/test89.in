5
-194008456313 -490590244583 559318244245 -14996225805 -215445269420 -377710734943
195058742831 529098257661 -296194519055 220739761526 363700846652 -99025150840
-734644143105 417963214050 -152177228523 -844606291398 912681774229 435899071450
-525676329669 -944377438963 -495707818314 -582565250413 264214361455 -780785992804
-919613046357 -450001096667 -387947607902 -237274312344 -542644536736 637148771390
