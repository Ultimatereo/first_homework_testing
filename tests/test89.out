-0.0838861
1.35553
-0.121454
-1.34907
-1.47938
